modules/run-vault/run-vault

```diff
@@ -299,9 +300,14 @@ EOF
 )
     consul_storage_type="ha_storage"
     dynamodb_storage_type="ha_storage"
+    service_registration=$(cat <<EOF
+service_registration "consul" {
+  address = "127.0.0.1:8500"
```

Comment:
Are we sure that it's always going to be a local Consul agent at that exact port? Or should these items be configurable?

Reply:
Indeed, better with configurable address